app/views/group_offers/show.html.slim

```diff
@@ -68,23 +68,20 @@ h1= @group_offer.title
             td= t_attr(:schedule_details)
             td= @group_offer.schedule_details
 
-- if !@group_offer.external? && @group_offer.responsible
-  - if @group_offer.volunteer_responsible_state?
-    h3= t("volunteer_responsible_state.#{@group_offer.volunteer_responsible_state}")
-  - else
-    h3= t_attr(:responsible)
-  .row
-    .col-xs-12
-      ul
-        li= link_to @group_offer.responsible, volunteer_path(@group_offer.responsible)
-
-- if @group_offer.volunteers.any?
+- if @group_offer.group_assignments.any?
   h3= t_attr(:volunteers)
   .row
-    .col-xs-12
-      ul
-        - @group_offer.volunteers.each do |volunteer|
-          li= link_to volunteer, volunteer_path(volunteer)
+    ul
+      - @group_offer.group_assignments.each do |group_assignment|
+        li
+          .col-xs-12.col-md-3
+            = link_to group_assignment.volunteer, volunteer_path(group_assignment.volunteer)
+          .col-xs-12.col-md-3
+            = t_attr(:responsible, GroupAssignment) if group_assignment.responsible
```

Comment:
Wouldn't it be better to display `.col-xs-12.col-md-3` on `if group_assignment.responsible == false`?
I mean if above `.col-xs-12.col-md-3`

Reply:
No because there will be no consistency in the contents of the columns.